src/puppetlabs/trapperkeeper/services/metrics/metrics_core.clj

```diff
@@ -1,6 +1,9 @@
 (ns puppetlabs.trapperkeeper.services.metrics.metrics-core
-  (:import (com.codahale.metrics JmxReporter MetricRegistry))
+  (:import (com.codahale.metrics JmxReporter MetricRegistry)
+           com.fasterxml.jackson.core.JsonParseException)
```

Comment:
I think for consistency we should change this to `(com.fasterxml.jackson.core JsonParseException)`